src/AuthFetcher.ts

```diff
@@ -46,87 +40,33 @@ export default class AuthFetcher extends EventEmitter {
     private authenticatedFetcher: IAuthenticatedFetcher,
     @inject("environmentDetector")
     private environmentDetector: IEnvironmentDetector
-  ) {
-    super();
-  }
-
-  private async loginHelper(
-    options: ILoginInputOptions,
-    localUserId?: string
-  ): Promise<ISolidSession> {
-    const internalOptions: ILoginOptions = validateSchema(
-      loginInputOptionsSchema,
-      options
-    );
-    if (localUserId) {
-      internalOptions.localUserId = localUserId;
-    }
-    return this.loginHandler.handle(internalOptions);
-  }
-
-  async login(options: ILoginInputOptions): Promise<ISolidSession> {
-    return this.loginHelper(options, this.globalUserName);
-  }
-
-  async fetch(url: RequestInfo, init?: RequestInit): Promise<Response> {
-    this.emit("request", url, init);
-    return this.authenticatedFetcher.handle(
-      // TODO: generate request credentials separately
-      {
-        localUserId: this.globalUserName,
-        type: "dpop"
-      },
-      url,
-      init
-    );
-  }
-
-  async logout(): Promise<void> {
-    await this.logoutHandler.handle(this.globalUserName);
-  }
-
-  async getSession(): Promise<ISolidSession | null> {
-    return this.sessionCreator.getSession(this.globalUserName);
-  }
-
-  async uniqueLogin(options: ILoginInputOptions): Promise<ISolidSession> {
-    return this.loginHelper(options);
-  }
+  ) {}
 
-  async onSession(
-    callback: (session: ISolidSession) => unknown
-  ): Promise<void> {
-    // TODO: this should be updated to handle non global as well
-    const currentSession = await this.getSession();
-    if (currentSession) {
-      callback(currentSession);
-    }
-    this.on("session", callback);
+  async login(sessionId: string, options: ILoginInputOptions): Promise<void> {
+    throw new Error("Inner not implemented");
```

Comment:
Just a niggle, but maybe just replace 'inner' in all these 'not implemented yet' exception messages with the name of the method.

Reply:
Resolved in d2176c38b94e9096f146cdb1246646bae23a2667